generators/app/generator.spec.ts

```diff
@@ -165,4 +165,110 @@ describe(`generator - ${generator}`, () => {
       });
     });
   });
+  describe('questions', () => {
+    describe('without answers', () => {
+      before(async () => {
+        await helpers.run(generatorPath).withSkipWritingPriorities();
+      });
+
+      it('should match order', () => {
+        expect(runResult.askedQuestions.map(({ name }) => name)).toMatchInlineSnapshot(`
```

Comment:
Locally running `npm test`, the `askedQuestion` returns undefined. Any clue?

Reply:
`npm ci`?
Requires an updated version of yeoman-test.